pkg/apis/eventing/v1/trigger_types.go

```diff
@@ -169,10 +169,10 @@ type SubscriptionsAPIFilter struct {
 	// +optional
 	Suffix map[string]string `json:"suffix,omitempty"`
 
-	// SQL is a CloudEvents SQL expression that will be evaluated to true or false against each CloudEvent.
+	// CESQL is a CloudEvents CESQL expression that will be evaluated to true or false against each CloudEvent.
 	//
 	// +optional
-	SQL string `json:"sql,omitempty"`
+	CESQL string `json:"cesql,omitempty"`
```

Comment:
This feels like a really confusing rename. Why not change `Suffix` to `CESuffix` and `Exact` to `CEExact`? Because those are all worse names and this filter is in the context of cloudevents filters. This is a docs problem, not an API naming problem.

Reply:
The feedback we've had so far is related to the strong database related connotation that comes with `sql`. What I read from the feedback is that calling it `cesql` makes it instantly obvious that this is a custom expression language related to CloudEvents.

Reply:
I agree with @devguyio - renaming to CESQL will help to avoid lot of user confusion and may help with google search in future ...

Reply:
I also think it's beneficial to change the name